packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.html

```diff
@@ -77,75 +281,6 @@ <h1 data-translate="pci_projects_project_serving_namespace_models_add"></h1>
                     </div>
                 </div>
             </div>
```

Comment:
missing "data-" prefix for a lot of custom attributes

Reply:
@radireddy see my following comment: https://github.com/ovh/manager/pull/3652#discussion_r491824430 ;)